src/controllers.php

```diff
@@ -2,23 +2,50 @@
 
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpFoundation\JsonResponse;
-use Symfony\Component\HttpFoundation\RedirectResponse;
-use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
-
-//Request::setTrustedProxies(array('127.0.0.1'));
+use Symfony\Component\HttpFoundation\Session\Session;
+use Silex\Provider\FormServiceProvider;
 
 $app->get('/', function () use ($app) {
-    return $app['twig']->render('index.html.twig');
+
+        if (($app['session']->get('is_logged')) === 1) {
+            var_dump($app['session']->isStarted());
+            return $app['twig']->render('index.html.twig');
+        } else {
+            return $app['twig']->render('login.html.twig');
+        }
 })
 ->bind('index');
 
 $app->get('/login', function () use ($app) {
     return $app['twig']->render('login.html.twig');
+
 })
+
 ->bind('login');
 
+$app->post('/login', function (Request $request) use ($app) {
+
+    //    $login = false;
+    $name = $request->get('name');
+    $password = $request->get('password');
+    $user = $app['db']->fetchAssoc('SELECT * FROM users WHERE name = ?', [$name]);
+
+    if (password_verify($password, $user['password'])) {
+//        die(var_dump($user['password']));
+        if (($app['session']->get('is_logged')) != 1) {
+            $app['session']->set('is_logged', 1);
+            $app['session']->set('user', ['name' => $name]);
+            $app['session']->start();
+            return $app->redirect('/');
+        }
+    } else {
+        echo 'Please check your input';
```

Comment:
powinienes zwrocic Response albo wyrenderowac templatke a nie uzywac echo (to moze byc uzyte ostatczenie tylko podczas debugu)